src/kili/adapters/kili_api_gateway/model_configuration/mappers.py

```diff
@@ -1,11 +1,111 @@
 """GraphQL payload data mappers for api keys operations."""
 
-from kili.domain.project_model import ProjectModelFilters
+from typing import Dict
```

Comment:
Shouldn't we move every part of `kili_api_gateway` related to llm to `llm/kili_api_gateway` ?

Reply:
Not sure about this, because all `kili_api_gateway` related code is currently in `kili_api_gateway`

I suggest to rename folder `adapters/kili_api_gateway/model_configuration` -> `adapters/kili_api_gateway/llm`